### DISCUSSION TITLE
Extended/"advanced" wildcard support

### DISCUSSION THREAD

**Discussion by yo1nkers** [mcmonkeyprojects/SwarmUI#465] [Ideas]
I see other user created wildcards such as Billions-of-*, a series of wildcards which have heavy randomizers and emphasis on creative generation. These are .yaml wildcards, and are structured differently rather than just one tag on each line.

Happy to give further example if required, but I'd love to see the ability to use these stronger wildcards. I've moved to SwarmUI permanently but one of the things I used to love using was Dynamic Prompts, it had wonderful functionality.

**Comment by mcmonkey4eva** [comment]
That advanced usage isn't unsupported. In fact Swarm supports a much more advanced range. That doesn't mean pre-existing yaml files made for auto work in swarm though. There was discussion about an importer on discord, not sure if anyone actually made one
